asf/common/components/wifi/winc1500/driver/source/nmspi.c

```diff
@@ -42,7 +42,7 @@
 
 #ifdef CONF_WINC_USE_SPI
 
-#define USE_OLD_SPI_SW
+//#define USE_OLD_SPI_SW
```

Comment:
Zephyr has a policy of using classic `/* */` comments. It's arguable whether that should apply to this repo, but to be on the safe side, I'd suggest to follow it.

Also, please add a comment before this define along lines of `/* Don't force USE_OLD_SPI_SW, instead we pass it from Zephyr side. */`

Reply:
Done.